platform-sdk/swirlds-config-extensions/src/test/resources/test.yaml

```diff
@@ -1,5 +1,5 @@
 gossip:
-  interfaceBindings:
+  networkEndpoints:
     - { "nodeId": 0, "hostname": "10.10.10.1", "port": 1234 }
```

Comment:
This format seems too hacky for my taste. As it is already in the repo, this comment is not a blocker,
but I'm curious why we didn't use a complete yaml list like:
```
networkEndpoints:
  - nodeId: 0
    hostname: 10.10.10.1
    port: 1234
  - nodeId: 0
    hostname: 10.10.10.2
    port: 1234
```